src/lib/integration_test_lib/intf.ml

```diff
@@ -254,6 +253,23 @@ module Engine = struct
     val genesis_keypairs : t -> Network_keypair.t Core.String.Map.t
 
     val initialize_infra : logger:Logger.t -> t -> unit Malleable_error.t
+
+    val id : t -> string
+
+    (*** [alias] is instantiated when command line args are parsed *)
+    val alias : (string * string) option ref
+
+    (*** [archive_image] is instantiated when command line args are parsed *)
+    val archive_image : string option ref
+
+    (*** [config_path] is instantiated when command line args are parsed *)
+    val config_path : string ref
+
+    (*** [keypairs_path] is instantiated when command line args are parsed *)
+    val keypairs_path : string ref
+
+    (*** [mina_image] is instantiated when command line args are parsed *)
+    val mina_image : string ref
   end
```

Comment:
@Isaac-DeFrain Why have these been added to the intf? Weren't the archive image and mina_image already being passed in to the cli for the old gcp engine?

Reply:
These values are used in the `network_config` module, but their values are set in the `test_executive` module when the command line args are parsed. Thus, we need a way to access them in the `test_executive`, e.g. via `Engine.Network.keypairs_path`

Reply:
I'm not entirely sure, but they definitely weren't using these values higher up in the dependency DAG as we are in the abstract engine